enum.go

```diff
@@ -97,6 +98,12 @@ func (e Enum[M, V]) Members() []M {
 	return e.members
 }
 
+// Random returns a randomly selected member from the enum.
+func (e Enum[M, V]) Random() M {
```

Comment:
Let's make it accept a random seed, like here:

https://github.com/life4/genesis/blob/master/slices/slice.go#L15

It might be useful for testing code relying on this method.

Reply:
Also, maybe call it `Choice` instead? Then it would match the naming of genesis and [Python's random module](https://docs.python.org/3/library/random.html#random.choice).

Reply:
I renamed the function to Choice :+1:

After `rand.Seed` became deprecated (since Go 1.20), I think the "global seed" can be trusted to be set to something that will result in random numbers, and I suspect that `seed = time.Now().UnixNano()` (like in genesis) is needed anymore.

Reply:
Genesis uses the UNIX time because there is no way to get the global source, and to have predictable behavior it uses a locally created rand.Rand instance regardless of if the seed is user-provided or generated.